package.json

```
@@ -6,7 +6,10 @@
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "cy:open": "cypress open",
+    "test": "cypress run",
+    "test:all": "start-test start:frontend http://localhost:3000 start:server http://localhost:3001 test"
```

What about calling the script `start:servers:and:test`?

This way, it would be more descriptive.

```suggestion
    "start:servers:and:test": "start-test start:frontend http://localhost:3000 start:server http://localhost:3001 test"
```

---

Agree, love it!

---

Done